app/src/main/java/com/auroralabs/tendarts/app/viewholders/LogViewHolder.java

```diff
@@ -0,0 +1,55 @@
+package com.auroralabs.tendarts.app.viewholders;
+
+import android.support.v7.widget.RecyclerView;
+import android.view.View;
+import android.widget.TextView;
+
+import com.auroralabs.tendarts.R;
+import com.auroralabs.tendarts.domain.entities.LogEntity;
+
+import butterknife.BindView;
+import butterknife.ButterKnife;
+
+/**
+ * Created by luismacb on 12/2/18.
+ */
+
+public class LogViewHolder extends RecyclerView.ViewHolder {
+
+    public static final int LAYOUT_RESOURCE_ID = R.layout.view_holder_log;
+
+    private LogEntity logEntity;
+
+    @BindView(R.id.log_text)
+    TextView logText;
+
+    public LogViewHolder(View itemView) {
+        super(itemView);
+
+        ButterKnife.bind(this, itemView);
+    }
+
+    public void bind(LogEntity logEntity) {
+
+        this.logEntity = logEntity;
+
+        if (logEntity.getCategory() == null) {
+            logEntity.setCategory("");
+        }
+
+        if (logEntity.getType() == null) {
+            logEntity.setType("");
+        }
+
+        if (logEntity.getMessage() == null) {
+            logEntity.setMessage("");
+        }
+
+        String message = String.format("%s, category: '%s', type: '%s', message: '%s'",
```

Comment:
It's a little bit ogg using String.format(), isn't I? You could use StringBuilder class methods